client/src/includes/inlinePanel.js

```diff
@@ -117,6 +121,16 @@ function inlinePanel(opts) {
     }
   };
 
+  /**
+   * Adds the child’s count next to its heading label, ignoring deleted items.
+   */
+  self.updateChildCount = function updateChildCount() {
+    const forms = self.formsElt.children(':not(.deleted)');
+    forms.each(function updateCountState(i) {
+      $('[data-inline-panel-child-count]', this).text(` ${i + 1}`);
```

Comment:
This will match all `[data-inline-panel-child-count]` elements inside the form, including any nested ones. While the level of support for nested InlinePanels is somewhat ambiguous (and I suspect there are similar issues elsewhere, e.g. selecting the up/down controls), we should avoid knowingly introducing new failure cases - and in this case I think we can reasonably do that by just selecting the first match:
```suggestion
      $('[data-inline-panel-child-count]', this).first().text(` ${i + 1}`);
```

Reply:
Good point, I keep forgetting about nesting. Will adjust!